ofl/notoemoji/METADATA.pb

```diff
@@ -0,0 +1,68 @@
+name: "Noto Emoji"
+designer: "Google"
+license: "OFL"
+category: "SANS_SERIF"
```

Comment:
Is this how emoji are categorized?

Reply:
Viv recommends "DISPLAY"

Reply:
We discussed this further and recommend adding a new category called "Symbols" that captures families like Noto Emoji, Emoji BW, Symbols, Symbols 2.

Shouldn't block landing this PR IMO.

Reply:
SGTM